packages/app/src/AuthPage.tsx

```diff
@@ -90,10 +134,17 @@ export default function AuthPage({ action }: { action: 'register' | 'login' }) {
                 <Form.Control
                   data-test-id="form-password"
                   id="password"
-                  name="password"
                   type="password"
                   className="border-0"
+                  {...form.password}
```

Comment:
nit: not super critical. I think it would be nice to add 'confirm password' field and the icon to toggle 'show password'

Reply:
there's #40 which adds the toggle, perhaps I shouldn't duplicate it
I'm down to add the password confirmation field, but not sure if I get to it today or tomorrow, feel free to go ahead and merge it, and I can implement it sometime this week with a follow-up PR. This will also give you some time figure out if you proceed with #40 or not. How does that sound?